python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -777,6 +781,120 @@ def median(self):
         """Get the column-wise median of the values in each group."""
         return self.agg("median")
 
+    def corr(self, method="pearson", min_periods=1):
+        """
+        Compute pairwise correlation of columns, excluding NA/null values.
+
+        Parameters
+        ----------
+        method: {"pearson" (default), "kendall", "spearman"} or callable
```

Comment:
I think this is our normal syntax for this kind of doc
```suggestion
        method: {"pearson", "kendall", "spearman"} or callable, default "pearson"
```